components/dialog/CandidateCallout.tsx

```diff
@@ -1,27 +1,66 @@
-import { Callout, DataList } from '@radix-ui/themes'
+import { shortenEmail } from '@/lib/utils'
+import { Callout, DataList, Flex } from '@radix-ui/themes'
 import React, { FC } from 'react'
 
 interface CandidateCalloutProps {
   firstName: string
   surname: string
   ucasNumber: string
+  showExtraInformation?: boolean
+  reviewer?: string
+  reviewerPercentile?: number | null
+  overallScore?: number | null
+  academicComments?: string | null
 }
 
-const CandidateCallout: FC<CandidateCalloutProps> = ({ firstName, surname, ucasNumber }) => {
+const CandidateCallout: FC<CandidateCalloutProps> = ({
+  firstName,
+  surname,
+  ucasNumber,
+  showExtraInformation = false,
+  reviewer,
+  overallScore,
+  reviewerPercentile,
+  academicComments
+}) => {
   return (
     <Callout.Root>
-      <DataList.Root>
-        <DataList.Item align="center">
-          <DataList.Label>Applicant:</DataList.Label>
-          <DataList.Value className="font-bold">
-            {firstName} {surname}
-          </DataList.Value>
-        </DataList.Item>
-        <DataList.Item align="center">
-          <DataList.Label>UCAS number:</DataList.Label>
-          <DataList.Value className="font-bold">{ucasNumber}</DataList.Value>
-        </DataList.Item>
-      </DataList.Root>
+      <Flex align="center" gap="9">
+        <DataList.Root>
+          <DataList.Item align="center">
+            <DataList.Label>Applicant:</DataList.Label>
+            <DataList.Value className="font-bold">
+              {firstName} {surname}
+            </DataList.Value>
+          </DataList.Item>
```

Comment:
Considering the rate of duplication, how about extracting a DataListItem component with label and value props?